api/src/main/java/cn/nukkit/api/util/Callback.java

```diff
@@ -0,0 +1,7 @@
+package cn.nukkit.api.util;
+
+public interface Callback<T> {
```

Comment:
`java.function.Consumer`?

Reply:
I'll look into ```java.function.Consumer```, its a simple callback though that is intended to allow us to use callbacks if we need to use them.

Reply:
Consumer does literally exactly the same thing as that, and doesn't require a new class.

Reply:
ok, removed